src/reader-activation/index.js

```diff
@@ -159,6 +159,7 @@ export function openAuthModal( config = {} ) {
 			},
 			content: null,
 			trigger: null,
+			skipClose: false,
```

Comment:
This is a nonblocking nit, but the name `skipClose` feels a bit strange to me. It's not really something that is being skipped, but more like something that is just not being done. It might be better to call this something like `closeOnSuccess` or something along those lines with a default value of `true`.

Reply:
Ah thanks @chickenn00dle! I am notoriously bad at naming stuff, this is much nicer 😆 It has been updated in [4ec56a9](https://github.com/Automattic/newspack-plugin/pull/3369/commits/4ec56a97e5ccca5899743a7123f4403b4c2cf897)!